docs/section15/Rule15-9.md

```diff
@@ -0,0 +1,28 @@
+# Transformers - Rule 15-9
+**Rule ID:** Rule 15-9  
+**Rule Description:** Transformer capacity ratio reported in Baseline RMR equals User RMR  
+**Rule Assertion:** User RMR = Baseline RMR  
```

Comment:
For envelope and lighting, we express such rules as B-RMR=P-RMR, and also have rule that P-RMR=U-RMR. Even though it's mathematically the same, I would stick with the existing framework throughout the RDSs for consistency, in stead of doing U-RMR = B-RMR.